recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -47,8 +47,20 @@ hashalg="sha1"
 good_pcrs="/config/good.pcrs"
 bad_pcrs="/boot/system/tpm/bad.pcrs"
 root_dev="/dev/mapper/xenclient-root"
-
-while getopts ":sufc:t:p:g:b:r:" opt; do
+# First file is parsed and signed with pesign, following object with extend the
+# hash.
```

Comment:
This comment perhaps needs a bit more explanation from my end. According to the UEFI specification PCR4 should contain measurement of UEFI executables using the Authenticode hash, which is what pesign produces. Since the shim gets loaded by the firmware directly it gets measured as Authenticode. Due to a firmware bug on Dell systems the other executables (xen/dom0 kernel) can't be measured with Authenticode, so they are measured as raw buffers. Ideally once Dell fixes their issue (they have been notified) and all firmwares uniformly can measure with Authenticode we would make this UEFI spec compliant. But it's arguably a low priority.